Code_of_conduct.md

```diff
@@ -0,0 +1,79 @@
+---
+Name: Code of Conduct
+Description: A general Policy about the code of conduct in the null community
+Status: Draft
+---
+
+## Overlaying Objective
+
+Refrain yourself from engaging with anyone or posting any content which can hurt anyone's sentiments.
+
+## GEOPOLITICAL ASPECTS
+1. You must consider the geopolitical aspects before posting any message. 
+1. You have to be careful while forwarding URLs. You must not forward any URL without reading the content yourself and having understood its implications.
+1. Must ensure that your post does not contain any sensitive information about any country or person. No matter even if your post has already been published and is in the public domain.
+1. Your post must not hurt the sentiments of any country or person.
+
+## PRIVACY
+1. Don't approach any new member for their personal information immediately after joining the group unless you know him/her personally. 
+1. Members can choose to share their personal information in the group or privately with someone at their own risk.
+1. No use of contacts from this Group to use as a business lead.
+1. Do not share the contact details and profile pictures of the members with anyone who is not a member of this group and without the permission of the member. 
+
+## DISCUSSION
+No Discussion & Discrimination based on 
+* politics, 
+* religion, 
+* region, 
+* organization, 
+* industry, 
+* Sector, or 
+* any other similar criteria;
+
+Do not criticize any organization or service provider or Government organizations and regulators.
```

Comment:
This line needs to be removed. We are open to discussion or debate however we do not accept non-parliamentary language.